src/OpenLoco/src/Localisation/Formatting.cpp

```diff
@@ -217,7 +340,7 @@ namespace OpenLoco::StringManager
         const char* prefixSymbol = getString(currency->prefixSymbol);
         buffer = formatStringPart(buffer, prefixSymbol, nullptr);
 
-        buffer = formatInt48Grouped(localisedValue, buffer, currency->separator);
```

Comment:
Not sure what to do with this. #2118 treats it as `value * static_cast<uint64_t>(std::pow(10, separator))`??